src/main/java/com/github/britter/springbootherokudemo/entity/enumeration/RoomStatus.java

```diff
@@ -0,0 +1,5 @@
+package com.github.britter.springbootherokudemo.entity.enumeration;
+
+public enum RoomStatus {
+    AVAILABLE, OCCUPIED, UNKNOWN
```

Comment:
Czy jest jakaś różnica funkcjonalna w porównaniu do tego, co było wcześniej? Bo ja jej nie widzę.
Wcześniej Boolean mógł mieć wartość True/False/null i sprowadzało się to do tego samego, co w przypadku tego enuma.
Żeby wprowadzić funkcjonalność "wietrzenia" potrzebujemy chyba jeszcze czwartej wartości enuma (np. AIRING - wg. Google Translate "wietrzenie")?